corehq/apps/app_manager/management/commands/helpers.py

```diff
@@ -33,6 +36,9 @@ class AppMigrationCommandBase(BaseCommand):
     chunk_size = 100
     include_builds = False
     include_linked_apps = False
+    # Overwrite these to avoid conflicting naming with another managment command.
```

Comment:
if you want to force these to be overwritten then you should leave them as `None` here and then validate that they are not `None` when running the command.

Alternatively you could use `self.__class__.__name__` to make them unique per class.